packages/ckermit.rb

```diff
@@ -0,0 +1,19 @@
+require 'package'
+
+class Ckermit < Package
+  description 'C-Kermit is a combined serial and network communication software package.'
+  homepage 'http://www.kermitproject.org/ck90.html'
+  version '8.0.211'
+  source_url 'https://fossies.org/linux/privat/old/cku211.tar.gz'
+  source_sha256 '0503518ef8f225d06749f7923ae84521bc358daf353222c75c009cd040dc5e26'
+
+  depends_on 'ncurses'
+
+  def self.build
+    system 'make KFLAGS="-DCK_NCURSES -I/usr/local/include/ncurses" LNKFLAGS="-lresolv -lcrypt" linux'
+  end
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
```

Comment:
If this package installs any library files, those should be abstracted also.  Something like:
```
system 'make', "DESTDIR=#{CREW_DEST_DIR}", "LIBDIR=#{CREW_LIB_PREFIX}", 'install'
```
This may or may not work.  Examine `Makefile` and look for the `LIBDIR` keyword.